docs/sdks/sdk-ref-python.md

```diff
@@ -185,6 +186,35 @@ If you're using server-only integration (see [Publisher Integration Guide, Serve
 
    If the user has opted out, this method returns `None`, indicating that the user's identity should be removed from the session. To confirm optout, you can use the `token_refresh_response.is_optout()` function.
 
+## Usage for Advertisers and Data Providers
+1. Create an instance of IdentityMapClient as an instance variable.
+   ```py
+   client = IdentityMapClient(base_url, auth_key, secret_key)
+   ```
+
+2. Call a function that takes email addresses or phone numbers as input and generates an IdentityMapResponse object. The following example uses email addresses:
+   ```py
+   identity_map_response = client.generate_identity_map(IdentityMapInput.from_emails(["email1@example.com", "email2@example.com"]))
+   ```
+
+>Note: The SDK hashes input values before sending them. This ensures that raw email addresses and phone numbers do not leave your server.
```

Comment:
Same here. No mentions of phone numbers anywhere.

My other small comments on the UID2 version are also applicable, and otherwise looks good!

Reply:
Removed the phone number